### ISSUE TITLE
Import of CSV backup crashes/fails

### ISSUE THREAD

**Issue by GoogleCodeExporter** [evancharlton/android-mileage#22]
```
What steps will reproduce the problem?
1. Install Mileage for Android -- I also renamed the default vehicle and 
applied the following details...

Name: Suzy
Make: Subaru
Model: Outback Sport
Year: 1999

2. Add two test fillups/entries. In this case:
       3/5/2011, 4 gallons, $4.00
       3/1/2011, 3 gallons, $3.00 (added the comment "Comment is hree")

3. Press Menu button on phone
4. Choose Import / Export
5. Choose Export Data
6. Select "Spreadsheet" from drop down
7. Filename: mileage-export.csv
8. Click Export data.

9. Screen displays:
       exporting to mileage-export.csv ...
       Wrote headers...
       2 rows written (2 total)
       Exported data to mileage-export.csv!

10. Delete the two entries manually (longpress > Delete > confirm).

11. Press Menu button on phone

12. choose Import / Export
13. choose Import Data
14. Select "Spreadsheet" from drop down. Select "mileage-export.csv" from drop 
down. Leave "Erase database before importing" unchecked.
15. Press Import data.

16. Press next at the Column Mapping screen.
17. Leave setting at "1" for vehicle mapping on Suzy, press next.
18. "Set date format" screen. Shows...
       Saved Date: 03/05/2011
       Date format: YYYY-MM-DD HH:MM:SS
       Parsed date: Could not parse date!

From this step there are a number of scenarios. 
Scenario one - Press next, and it reports...
       Erased statistics cache
       0 rows processed (2 total)
       Processed 0 rows!

Scenario two - from the "Date format" drop down, select each of the date format 
options. 

In the case of options 1-7, the "Could not parse Date!" does not change. 

In the case of options 8-12, the error "Sorry! - The application Mileage 
(process com.evancharlton.mileage) has stopped unexpectedly. Please try again." 
appears, and gives the option of "force close" and "report". Upon pressing 
"Force Close" you are kicked back to the "Column mappings" screen where you can 
continue attempting to import.



Expected behavior: I expect the default selection (as the export options were 
all default for that filetype) to function normally and parse the date. If not 
that, then at least one of the 12 options to parse the date without crashing.

Actual behavior: Unable to parse date. Import process/program crashes.

Mileage version: 3.0.5
Android version: 2.2.1
Device: HTC Glacier / MyTouch 4G

Please provide any additional information below.

Kernal version: 2.6.32.21-g899d047
htc-kernel@and18-2 #1
Tue Oct 26 16:10:01 CST 2010
```

Original issue reported on code.google.com by `lilnig...@gmail.com` on 5 Mar 2011 at 8:19

**Comment by GoogleCodeExporter** [comment]
```
Also, I did not clarify, no changes are made to the spreadsheet/csv inbetween 
export and import.
```

Original comment by `lilnig...@gmail.com` on 5 Mar 2011 at 8:20

**Comment by GoogleCodeExporter** [comment]
```
http://code.google.com/p/android-mileage/source/detail?r=1f32b4992ddf1f0551f6eb0
9c28bd19b005e5d49

Will be included in 3.1.0
```

Original comment by `evancharlton` on 17 Apr 2011 at 8:20
- Changed state: **Fixed**
